scheduler/cloudformation/template.go

```diff
@@ -690,6 +690,28 @@ func (t *EmpireTemplate) addService(tmpl *troposphere.Template, app *twelvefacto
 		"ServiceName":    fmt.Sprintf("%s-%s", app.Name, p.Type),
 		"ServiceToken":   t.CustomResourcesTopic,
 	}
+	if v := p.ECS; v != nil {
+		if len(v.PlacementConstraints) > 0 {
+			var placementConstraints []interface{}
+			for _, c := range v.PlacementConstraints {
+				placementConstraints = append(placementConstraints, map[string]interface{}{
+					"Type":       c.Type,
+					"Expression": c.Expression,
+				})
+			}
+			serviceProperties["PlacementConstraints"] = placementConstraints
```

Comment:
This should actually be added to the task definition, rather than the service. Reason being, this isn't updatable on a service, which would require a replacement of the service. If we do it on the task definition, then no replacement required. Placement strategies can only be applied to the service, so that would need to remain.